packages/api/helper/sequelize.js

```diff
@@ -0,0 +1,7 @@
+const { Sequelize } = require('sequelize');
+const DATABASE_URL =
+  'postgres://ryanmansfield:postgres@localhost:5432/ryanmansfield';
```

Comment:
This should be available in the environment in Heroku, so should we consider pulling this from there (the process.env shenanigans). We could then use config.json if the config var isn't there.